includes/object-cache.php

```diff
@@ -2410,13 +2351,18 @@ public function increment( $key, $offset = 1, $group = 'default' ) {
         }
 
         try {
-            $value = (int) $this->parse_redis_response( $this->maybe_unserialize( $this->redis->get( $derived_key ) ) );
-            $value += $offset;
-            $result = $this->parse_redis_response( $this->redis->set( $derived_key, $this->maybe_serialize( $value ) ) );
+            if ( $this->use_igbinary ) {
+                $value = (int) $this->parse_redis_response( $this->maybe_unserialize( $this->redis->get( $derived_key ) ) );
+                $value += $offset;
+                $result = $this->parse_redis_response( $this->redis->set( $derived_key, $this->maybe_serialize( $value ) ) );
```

Comment:
@yatsukhnenko Can you add a call for `PTTL` and set it with `SET PX` if the key had an expiry? For `increment()` and `decrement()`.

Probably needs to be tested against Predis and PhpRedis, not sure if they take the same arguments in `set()`